Mathlib/MeasureTheory/Constructions/Polish.lean

```diff
@@ -886,8 +1006,8 @@ end PolishSpace
 
 namespace MeasureTheory
 
-variable (α)
-variable [MeasurableSpace α] [PolishSpace α] [BorelSpace α]
+variable (α : Type*)
```

Comment:
Why did you change that? There is already a variable alpha in scope, and you are introducing a second one -- use `#where` to see what is already defined at some place.

Reply:
Sorry about that. I was trying to get an `alpha` without a `TopologicalSpace` instance, but I see now that this isn't a good way to do that.